packages/ts/react-crud/src/autoform.tsx

```diff
@@ -198,9 +200,11 @@ export function ExperimentalAutoForm<M extends AbstractModel>({
   }
 
   return (
-    <section className="flex flex-col p-m gap-m">
-      {layout}
-      {formError ? <div style={{ color: 'var(--lumo-error-color)' }}>{formError}</div> : <></>}
+    <VerticalLayout className="auto-form" theme="spacing">
```

Comment:
Some opinionated changes here:
- Let's not use utility classes in components. There is no guarantee that those are loaded in an application which silently breaks things. Let's only assume that Lumo CSS properties are there.
- Don't add any sort of margin or padding to the outer element of a component. That is easy enough to add on the application-side, plus it requires customization to remove it if you want to do anything practical, for example show a heading and the form with the same horizontal alignment.
- Using `section` doesn't provide any benefit here as it's lacking a heading.

Also restored the classes that are required for properly styling the component when used in auto crud.